packages/slate-editor/src/modules/editor-v4-story-bookmark/components/StoryBookmarkElement/StoryBookmarkElement.tsx

```diff
@@ -28,6 +31,10 @@ export function StoryBookmarkElement({ attributes, children, element, params }:
         loadStory();
     }, [loadStory]);
 
+    if (error) {
+        console.log(error);
```

Comment:
@e1himself do you know already made solutions for handling errors?

Reply:
https://github.com/prezly/slate/blob/b9758a8db1016335e59f6beebfb7603a0017ab62/packages/slate-editor/src/modules/editor-v4-embed/lib/useFloatingEmbedInput.ts#L89-L93